pkgs/by-name/lu/ludusavi/package.nix

```
@@ -94,7 +109,7 @@ rustPlatform.buildRustPackage rec {
     homepage = "https://github.com/mtkennerly/ludusavi";
     changelog = "https://github.com/mtkennerly/ludusavi/blob/v${version}/CHANGELOG.md";
     license = licenses.mit;
-    maintainers = with maintainers; [ pasqui23 ];
+    maintainers = with maintainers; [ pasqui23 megheaiulian];
```

```suggestion
    maintainers = with maintainers; [ pasqui23 megheaiulian ];
```
Should be in a separate commit `ludusavi: add megheaiulian as maintainer`

Optionally make another commit `ludusavi: format with nixfmt` to ensure CI checks formatting for this file in the future

---

@donovanglover Thank you for the feedback. Please check again!